requirements.txt

```diff
@@ -1,38 +1,11 @@
-attrs==19.3.0
 biopython==1.77
-certifi==2020.6.20
-chardet==3.0.4
-coverage==5.2.1
 coveralls==2.1.1
-Cython==0.29.21
-DateTime==4.3
-decorator==4.4.2
-docopt==0.6.2
-idna==2.10
-iniconfig==1.0.1
-jsonschema==3.2.0
-more-itertools==8.4.0
-numpy==1.22.0
-packaging==20.4
-pandas==1.5.0
-parse==1.16.0
-pluggy==0.13.1
-progressbar2==3.51.4
-py==1.10.0
-pyparsing==2.4.7
-pyrsistent==0.16.0
-pytest==6.0.1
-pytest-cov==2.10.0
-pytest-runner==5.2
-python-dateutil==2.8.1
-python-utils==2.4.0
-pytz==2020.1
-requests==2.25.1
-requests-mock==1.8.0
-retry==0.9.2
-setuptools-git==1.2
-sh==1.13.1
-six==1.15.0
-toml==0.10.1
-urllib3==1.26.5
-zope.interface==5.1.0
\ No newline at end of file
+jsonschema
+jupyter
+numpy
+pandas
+pytest
+pytest-cov
+requests
+requests-mock
+retry
```

Comment:
Why are we changing the requirements to non specific version ?

Reply:
added them back, just a holdover from my clean-up...